src/sentry/conf/server.py

```diff
@@ -579,6 +579,9 @@ def SOCIAL_AUTH_DEFAULT_USERNAME():
     Queue("events.reprocess_events", routing_key="events.reprocess_events"),
     Queue("events.save_event", routing_key="events.save_event"),
     Queue("files.delete", routing_key="files.delete"),
+    Queue(
+        "group_owners.process_suspect_commits", routing_key="group_owners.process_suspect_commits"
+    ),
```

Comment:
I think we need to ask ops to set up some workers for this new queue once it's live.

Reply:
Alright, I'll ask James about this.

Reply:
It's great to know before-hand but we usually will over-provision the workers that consumer from newly-created queues so that we can observe that a new queue needs to be isolated to it's own worker.